src/ideas/until-school-restarts.md

```diff
@@ -0,0 +1,51 @@
+---
+title: " Until School Restarts: Organize an “Ageless Kids’ Club”"
+date: 2020-03-29
+archived: true
+author: Jutta Treviranus
+thumbnailImage: /media/school-closed.jpg
```

Comment:
This image is missing from `src/media/` directory which causes the build to fail when turning off the "archive" flag.

Reply:
@cindyli yes, I'm not sure what to do about that as the image was also missing from this PR: #356. Should I delete the `thumbnailImage` attribute altogether, do you think? I did wonder if it was worth trying to find the original, perhaps on Jutta's Medium blog.

Reply:
Found it! [656cdd4](https://github.com/inclusive-design/idrc/pull/635/commits/656cdd498bb60d8b8a63d4128711e5ce35b4e3d7)

Reply:
@greatislander I agree it's not worth finding the original image. Deleting the `thumbnailImage` attribute is a good workaround. If you do so, please also delete the line 8 where the same image is refered in the article content.

Reply:
ah ha, just noticed you've restored the image. Cool.

Reply:
Thanks to the Internet Archive :)